### DISCUSSION TITLE
Next.js/Webpack: Module not found: Can't resolve './build/Release/ecdh'

### DISCUSSION THREAD

**Discussion by imsanchez** [Web3Auth/Web3Auth#1327] [Help]
Hello friends! I've used older versions of Web3Auth for other projects on Next v12 with no issues. 

I'm building a web dapp (https://www.utopeon.com) on Next v13 with Web3Auth.

I encountered this error in my terminal during development:
```
./node_modules/@toruslabs/eccrypto/index.js
Module not found: Can't resolve './build/Release/ecdh' in ...
```

It appears others have raised this similar issue on the repository which appears to be no longer maintained: https://github.com/torusresearch/eccrypto/issues/28. 

It happens when importing from `@web3auth/modal`.

I've tried adding some of the `@web3auth/*` repositories to `transpilePackages` in my project's `next.config.js` but no luck. I'll try again with `@toruslabs/eccrypto`.

Looking forward to some help finding a solution for this! Thank you so much, I appreciate everyone's support.

**Comment by tanguyenvn** [comment]
Hi, thanks for reporting this. We want to reproduce your issue on our end to further investigate.
We already have a Next example that integrates Web3Auth modal at https://github.com/Web3Auth/examples/tree/main/web-modal-sdk/modular-examples/next-modular-web3auth-example. I changed the version of Next in package.json as following:
```"next": "13.1.6",```
However, I didn't found any issue like you said.
If I didn't configure Next v13 correctly, is it possible for you to create a PR of the above example? We'll really appreciate it.
Thanks

**Reply by imsanchez** [reply]
Hey! I'll get to this asap and follow-up here.

I forgot to also mention it's installed via pnpm on a monorepo w/ Node v18.13.0.
